### ISSUE TITLE
Include testimonials in "Convince Your Manager" page

### ISSUE THREAD

**Issue by mjang** [writethedocs/www#419]
Periodically, I see people talking about what they leaned from WtD.

I think it'd help if we collected such info in pages similar to http://www.writethedocs.org/conf/portland/2018/convince-your-manager/

**Comment by plaindocs** [comment]
Sounds good to me.

**Comment by erinkcochran87** [comment]
Here you go, @mjang!

I added a bit about my company's doc transformation and a link to a blog post I wrote on the subject. Feel free to edit as you see fit :)

---

In my annual review yesterday, my boss said this to me:

> If you’d told me when we hired for your role three years ago that we’d have our docs using Jekyll, be hosted in GitHub, contain pieces of automation, and be open-sourced, I wouldn’t have believed you. It’s gone so far above and beyond my expectations. **You’ve taught me that there’s a depth to documentation I didn’t know existed** and I believe we’re on the forefront of where docs are heading.

I owe a lot of my progress this past year to this community - before y’all, I had no idea docs-as-code was even a thing. It’s revolutionized the way I - and my company - approach documentation. And the coolest part is that it wound up showing others that docs are more than translating programmese or reactively creating FAQ-style docs.

Thanks to the extensive knowledge and support of the WTD community, I was able to get to the next level in my career (hello, **Senior** title!) and make more valuable contributions to my company. The big one was [overhauling our documentation process](https://blog.stitchdata.com/the-stitch-docs-are-now-open-source-71a191aef95a), which is now simple, powerful, and open to anyone. It's greatly increased my productivity and the quality and accuracy of our documentation.

I'm excited and proud to be part of such an awesome community.

TL:DR; Thanks for the past year, all! WTD ftw. :muscle:

**Comment by mjang** [comment]
While I don't have Erin's testimonial, what I've learned from WtD has made me a better writer. 

* I frequently apply the principles of "Writing for users who don't read"
* I've used the lessons from various talks on UI Text to become the "GoTo" person on the subject
* The skills I've learned w/r/t API tools allow me to set up and try alternatives in our own doc environment.
* The lessons I'm learning now on Developer Experience (DX) will help me lay out what we can do to improve our DX.

**Comment by plaindocs** [comment]
This is great stuff! I'm a little swamped by Prague CFP right now, but I'll get this in soon.

**Comment by ericholscher** [comment]
https://twitter.com/KatherinCo/status/1292854855128801281
